src/blockdata/script.rs

```diff
@@ -180,7 +180,9 @@ impl fmt::Display for Error {
             Error::NumericOverflow => "numeric overflow (number on stack larger than 4 bytes)",
             Error::BitcoinConsensus(ref _n) => "bitcoinconsensus verification failed",
             Error::UnknownSpentOutput(ref _point) => "unknown spent output Transaction::verify()",
-            Error::SerializationError => "can not serialize the spending transaction in Transaction::verify()",
+            Error::SerializationError => {
```

Comment:
Newlines always get brackets I believe. I always assumed this was a counter to how one can leave them off in C when there is a single line in the following block of code and how this was seen by some as a bad thing to do (since its so easy to come back later and add a new line without noticing the scope).

Reply:
I'm pretty sure in Rust you don't actually need the braces -- this is purely a rustfmt thing. I agree that it's silly and unnecessary.

Reply:
Now included, thanks.